airflow-core/src/airflow/api_fastapi/core_api/datamodels/dag_run.py

```diff
@@ -74,6 +74,7 @@ class DAGRunResponse(BaseModel):
     run_type: DagRunType
     state: DagRunState
     triggered_by: DagRunTriggeredByType | None
+    triggering_user: str | None
```

Comment:
Should we use GetUserDep to get the username? instead of asking user to get them? User can intentionally send "wrong" user names